dev/files/killConjurServer

```diff
@@ -10,6 +10,10 @@ kill_recurse() {
 }
 
 conjur_server_pid=$(pgrep -f 'ruby /usr/local/bin/conjurctl server')
-if [ -n "$conjur_server_pid" ]; then
-  kill_recurse "$conjur_server_pid"
-fi
+#Iterate over conjur server process
+while IFS= read -r pid
```

Comment:
Why is it a `while` and not an `if?`
is it supposed to change on runtime? if so, is there a possibility for endless while loop?